examples/shared-vpc/main.tf

```diff
@@ -0,0 +1,52 @@
+# Create service accounts which will be used to represent various components of the application.
+module "service_account" {
+  source = "../../modules/service-account"
```

Comment:
Module sources will need to be updated to reference the Terraform Registry.

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/60/commits/259c10b8e5c9bb5a1770fc1b4a5618bd63e3e4d7.